msg/ko_KR.euckr/cubrid.msg

```diff
@@ -1437,12 +1437,7 @@ $ LOADDB
 1353 CAS ���� ���� : %1$d 
 1354 CAS ���� ���� : %1$d, %2$s
 
-1355 "%4$s"���� %3$d��° �ٿ�� %1$s "%2$s" �(��) �߸�� ����Դϴ�.�
-1356 "%3$s ���� %2$d��° �ٿ�� ȣ��Ʈ �� "%1$s"�(��) �ʹ� ��ϴ�, ȣ��Ʈ ���� ���̴� 256�� ���� �۰ų� ���ƾ� �մϴ�.�
-1357 "%3$s" ���� %2$d��° �ٿ�� ȣ��Ʈ �� "%1$s"�(��) �̹� �ֽ�ϴ�. �ϳ��� ȣ��Ʈ ���(��) �ߺ��Ǿ� �ٸ� IP �ּҸ� ���� �� ���ϴ�. �
-1358 "%2$s" ���� %1$d��° �ٿ�� IP �ּҿ� ȣ��Ʈ �� �� �� �����ϰ� �ۼ��Ǿ�� �մϴ�.�
-1359 cubrid_hosts.conf ���Ͽ�� "%1$s"�(��) ã� �� ���ϴ�.�
-1360 user defined host�� ����ϴ� ���߿� ������ �߻��Ͽ���ϴ�.
```

Comment:
It seems like it is being erased by accident.

Reply:
Oh, thank you.